### DISCUSSION TITLE
Got "Killed" on back-up

### DISCUSSION THREAD

**Discussion by VermilianBIO** [spatie/laravel-backup#1527] [Q&A]
I've the v.7 installed on my project. the backup is actually working fine with db back up and can upload to remote destination
But when I tried to back up the whole project (roughly 5G in size as zip), after some time on artisan command, the command killed itself.

Can anyone tell me what may have been the issue that I'm facing? I've tried using `php -d` to increase memory limit and execution time already but to no effect

editted** The "Killed" prompt happens on file transfer to disk

```
Created zip containing 45589 files and directories. Size is 4.26 GB
Copying zip to disk named oss...
Killed
```

**Comment by PaolaRuby** [comment]
V7 has no support
Maybe making a smaller pieces, also look at the log
